configure.ac

```diff
@@ -157,6 +157,7 @@ CXXFLAGS_COMMON+=" -Wwrite-strings"
 CXXFLAGS_COMMON+=" -Wno-switch-default"
 CXXFLAGS_COMMON+=" -Wconversion"
 CXXFLAGS_COMMON+=" -Wno-psabi"
+CXXFLAGS_COMMON+=" -Wno-register"
```

Comment:
Do we really need this? I search codebase and did not find register storage class specifier.
ref: https://gcc.gnu.org/onlinedocs/gcc/C_002b_002b-Dialect-Options.html#index-Wno-register